From 77b0b0a343cc909270d0857d186319151e712cd8 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Wed, 22 Oct 2014 11:37:34 -0400
Subject: [PATCH 119/119] main: fix up spacing in previous commit to match
 coding style

The previous commit introduced a get_splash_mode_from_mode function
which used the wrong indentation.

This commit fixes up the indentation.
---
 src/main.c | 38 +++++++++++++++++++-------------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/src/main.c b/src/main.c
index 101125b..2edc678 100644
--- a/src/main.c
+++ b/src/main.c
@@ -171,25 +171,25 @@ static bool sh_is_init (state_t *state);
 static ply_boot_splash_mode_t
 get_splash_mode_from_mode (ply_mode_t mode)
 {
-  ply_boot_splash_mode_t splash_mode;
-  switch (mode) {
-      case PLY_MODE_BOOT:
-          splash_mode = PLY_BOOT_SPLASH_MODE_BOOT_UP;
-          break;
-      case PLY_MODE_SHUTDOWN:
-          splash_mode = PLY_BOOT_SPLASH_MODE_SHUTDOWN;
-          break;
-      case PLY_MODE_UPDATES:
-          splash_mode = PLY_BOOT_SPLASH_MODE_UPDATES;
-          break;
-      default:
-          splash_mode = PLY_BOOT_SPLASH_MODE_INVALID;
-          break;
-  }
-
-  assert (splash_mode != PLY_BOOT_SPLASH_MODE_INVALID);
-
-  return splash_mode;
+        ply_boot_splash_mode_t splash_mode;
+        switch (mode) {
+                case PLY_MODE_BOOT:
+                        splash_mode = PLY_BOOT_SPLASH_MODE_BOOT_UP;
+                        break;
+                case PLY_MODE_SHUTDOWN:
+                        splash_mode = PLY_BOOT_SPLASH_MODE_SHUTDOWN;
+                        break;
+                case PLY_MODE_UPDATES:
+                        splash_mode = PLY_BOOT_SPLASH_MODE_UPDATES;
+                        break;
+                default:
+                        splash_mode = PLY_BOOT_SPLASH_MODE_INVALID;
+                        break;
+        }
+
+        assert (splash_mode != PLY_BOOT_SPLASH_MODE_INVALID);
+
+        return splash_mode;
 }
 
 static void
-- 
1.9.0

